bindings/python/src/lib.rs

```diff
@@ -0,0 +1,259 @@
+use anyhow::Result;
+use errors::*;
+use limbo_core::IO;
+use pyo3::prelude::*;
+use pyo3::types::PyList;
+use pyo3::types::PyTuple;
+use std::sync::{Arc, Mutex};
+
+mod errors;
+
+#[pyclass]
+#[derive(Clone, Debug)]
+struct Description {
+    #[pyo3(get)]
+    name: String,
+    #[pyo3(get)]
+    type_code: String,
+    #[pyo3(get)]
+    display_size: Option<String>,
+    #[pyo3(get)]
+    internal_size: Option<String>,
+    #[pyo3(get)]
+    precision: Option<String>,
+    #[pyo3(get)]
+    scale: Option<String>,
+    #[pyo3(get)]
+    null_ok: Option<String>,
+}
+
+impl IntoPy<Py<PyTuple>> for Description {
+    fn into_py(self, py: Python<'_>) -> Py<PyTuple> {
+        PyTuple::new_bound(
+            py,
+            vec![
+                self.name.into_py(py),
+                self.type_code.into_py(py),
+                self.display_size.into_py(py),
+                self.internal_size.into_py(py),
+                self.precision.into_py(py),
+                self.scale.into_py(py),
+                self.null_ok.into_py(py),
+            ],
+        )
+        .into()
+    }
+}
+
+#[pyclass]
+pub struct Cursor {
+    /**
+     * This read/write attribute specifies the number of rows to fetch at a time with .fetchmany().
+     * It defaults to 1 meaning to fetch a single row at a time.
+     */
```

Comment:
Btw, this is _not_ rustdoc style, see https://doc.rust-lang.org/rust-by-example/meta/doc.html for examples.